database/migrations/2024_03_26_135946_create_followers_table.php

```diff
@@ -12,7 +12,7 @@
     public function up(): void
     {
         Schema::create('followers', function (Blueprint $table) {
-            $table->unsignedInteger('user_id');
+            $table->foreignId('user_id')->constrained();
```

Comment:
[You can read the docs here to check how to link the foreign for follower_id](https://laravel.com/docs/10.x/migrations#column-method-foreignId:~:text=If%20your%20table%20name%20does%20not%20match%20Laravel%27s%20conventions%2C%20you%20may%20manually%20provide%20it)